src/boost_histogram/_internal/hist.py

```diff
@@ -772,6 +775,7 @@ def __getitem__(  # noqa: C901
                 assert isinstance(stop, int)
                 slices.append(_core.algorithm.slice_and_rebin(i, start, stop, merge))
 
+        logger.debug(f"Reduce with {slices}")
```

Comment:
```suggestion
        logger.debug("Reduce with %s", slices)
```
Easier to backport, and may be slightly optimized if unused.